### PR TITLE
Update v3 token management docs and add management tokens

### PR DESCRIPTION
**Pull request by sanderson**
This PR updates the `Manage tokens` sections in InfluxDB Cloud Dedicated and InfluxDB Clustered. It adds two new subsections:

- Database tokens
- Management tokens

It also updates all the links that point to database token-related content to point to the new path. I added redirect aliases as well, so there shouldn't be any links broken by this change.

- [x] Rebased/mergeable

### CODE REVIEW

**Review by gunnaraasen — APPROVED** [review]
LGTM!

File: content/influxdb/clustered/process-data/visualize/tableau.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Setting `useSystemTrustStore=false` is only necessary on macOS and doesn't actua
     
     - **Dialect**: PostreSQL
     - **Username**: _Leave empty_
-    - **Password**: [Database token](/influxdb/clustered/admin/tokens/)
+    - **Password**: [database token](/influxdb/clustered/admin/tokens/#database-tokens)
```
Comment (jstirnaman) [inline content/influxdb/clustered/process-data/visualize/tableau.md:90]:
```suggestion
    - **Password**: a [database token](/influxdb/clustered/admin/tokens/#database-tokens)
```

File: content/influxdb/clustered/query-data/execute-queries/client-libraries/go.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ In your `influxdb_go_client` module directory, create a file named `query.go` an
 Replace the following configuration values in the sample code:
 
 - {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: the name of the InfluxDB [database](/influxdb/clustered/admin/databases/) to query
```
Comment (jstirnaman) [inline content/influxdb/clustered/query-data/execute-queries/client-libraries/go.md:113]:
```suggestion
- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: the name of the [database](/influxdb/clustered/admin/databases/) to query
```

File: content/influxdb/clustered/query-data/execute-queries/client-libraries/go.md [hunk 3]
```diff
@@ -111,7 +111,7 @@ In your `influxdb_go_client` module directory, create a file named `query.go` an
...
-- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an InfluxDB [database token](/influxdb/clustered/admin/tokens/) with _read_ permission on the specified database
+- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an InfluxDB [database token](/influxdb/clustered/admin/tokens/#database-tokens) with _read_ permission on the specified database
```
Comment (jstirnaman) [inline content/influxdb/clustered/query-data/execute-queries/client-libraries/go.md:114]:
```suggestion
- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [database token](/influxdb/clustered/admin/tokens/#database-tokens) with _read_ permission on the specified database
```

File: content/influxdb/clustered/query-data/execute-queries/influxctl-cli.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ The `influxctl query` command only supports SQL queries; not InfluxQL.
 
 Provide the following with your command:
 
-- **Database token**: [Database token](/influxdb/clustered/admin/tokens/)
+- **Database token**: [database token](/influxdb/clustered/admin/tokens/#database-tokens)
```
Comment (jstirnaman) [inline content/influxdb/clustered/query-data/execute-queries/influxctl-cli.md:35]:
```suggestion
- **Database token**:  a [database token](/influxdb/clustered/admin/tokens/#database-tokens)
```

File: content/influxdb/clustered/query-data/execute-queries/influxctl-cli.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ The `influxctl query` command only supports SQL queries; not InfluxQL.
...
   with read permissions on the queried database. Uses the `token` setting from
```
Comment (jstirnaman) [inline content/influxdb/clustered/query-data/execute-queries/influxctl-cli.md:36]:
```suggestion
  with read permissions on the specified database. Uses the `token` setting from
```

File: content/influxdb/clustered/reference/client-libraries/flight/python-flightsql-dbapi.md [hunk 6]
```diff
@@ -94,7 +94,7 @@ client = FlightSQLClient(host='{{< influxdb/host >}}',
 
 Replace the following:
 
-- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an {{% product-name %}} [database token](/influxdb/clustered/admin/tokens/) with read permissions on the databases you want to query
+- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an {{% product-name %}} [database token](/influxdb/clustered/admin/tokens/#database-tokens) with read permissions on the databases you want to query
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/client-libraries/flight/python-flightsql-dbapi.md:97]:
```suggestion
- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [database token](/influxdb/clustered/admin/tokens/#database-tokens) with read permissions on the database you want to query
```

File: content/influxdb/clustered/reference/client-libraries/v2/python.md [hunk 7]
```diff
@@ -48,7 +48,7 @@ You'll need the following prerequisites:
     https://{{< influxdb/host >}}
     ```
 3. Name of the [database](/influxdb/clustered/admin/databases/) to write to.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/client-libraries/v2/python.md:50]:
```suggestion
3. The name of the [database](/influxdb/clustered/admin/databases/) to write to.
```

File: content/influxdb/clustered/reference/client-libraries/v2/python.md [hunk 8]
```diff
@@ -48,7 +48,7 @@ You'll need the following prerequisites:
...
-4. InfluxDB [database token](/influxdb/clustered/admin/tokens/) with permission to write to the database.
+4. InfluxDB [database token](/influxdb/clustered/admin/tokens/#database-tokens) with permission to write to the database.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/client-libraries/v2/python.md:51]:
```suggestion
4. A [database token](/influxdb/clustered/admin/tokens/#database-tokens) with permission to write to the database.
```

File: content/influxdb/clustered/reference/client-libraries/v3/go.md [hunk 9]
```diff
@@ -102,7 +102,7 @@ func main() {
 Replace the following configuration values:
 
 - {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: the name of the InfluxDB [database](/influxdb/clustered/admin/databases/) to query
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/client-libraries/v3/go.md:104]:
```suggestion
- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: the name of the [database](/influxdb/clustered/admin/databases/) to query
```

File: content/influxdb/clustered/reference/client-libraries/v3/go.md [hunk 10]
```diff
@@ -102,7 +102,7 @@ func main() {
...
-- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an InfluxDB [database token](/influxdb/clustered/admin/tokens/) with _read_ permission on the specified database
+- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an InfluxDB [database token](/influxdb/clustered/admin/tokens/#database-tokens) with _read_ permission on the specified database
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/client-libraries/v3/go.md:105]:
```suggestion
- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [database token](/influxdb/clustered/admin/tokens/#database-tokens) with _read_ permission on the specified database
```

File: content/influxdb/clustered/reference/client-libraries/v3/python.md [hunk 11]
```diff
@@ -222,7 +222,7 @@ client = InfluxDBClient3(token="DATABASE_TOKEN",
 Replace the following:
 
 - {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}: the name of your {{% product-name %}} [database](/influxdb/clustered/admin/databases/)
-- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an {{% product-name %}} [database token](/influxdb/clustered/admin/tokens/) with read permissions on the specified database
+- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an {{% product-name %}} [database token](/influxdb/clustered/admin/tokens/#database-tokens) with read permissions on the specified database
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/client-libraries/v3/python.md:225]:
```suggestion
- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [database token](/influxdb/clustered/admin/tokens/#database-tokens) with read permissions on the specified database
```

File: content/influxdb/clustered/reference/client-libraries/v3/python.md [hunk 12]
```diff
@@ -270,7 +270,7 @@ invokes the callback function, if specified, for the response status (success, e
...
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/client-libraries/v3/python.md:273]:
```suggestion
- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [database token](/influxdb/clustered/admin/tokens/#database-tokens) with read permissions on the specified database
```

File: content/influxdb/clustered/reference/sample-data.md [hunk 13]
```diff
@@ -156,7 +156,7 @@ Replace the following in the sample script:
 - {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}:
   your InfluxDB Clustered database
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/sample-data.md:157]:
```suggestion
  your [database](/influxdb/clustered/admin/databases/)
```

File: content/influxdb/clustered/reference/sample-data.md [hunk 14]
```diff
@@ -263,7 +263,7 @@ Replace the following in the sample script:
...
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/sample-data.md:264]:
```suggestion
  your [database](/influxdb/clustered/admin/databases/)
```

File: content/influxdb/clustered/reference/sample-data.md [hunk 15]
```diff
@@ -342,7 +342,7 @@ Replace the following in the sample script:
...
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/sample-data.md:343]:
```suggestion
  your [database](/influxdb/clustered/admin/databases/)
```

File: content/influxdb/clustered/reference/sample-data.md [hunk 16]
```diff
@@ -425,7 +425,7 @@ Replace the following in the sample script:
...
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/sample-data.md:426]:
```suggestion
  your [database](/influxdb/clustered/admin/databases/)
```

File: content/influxdb/clustered/reference/sample-data.md [hunk 17]
```diff
@@ -496,7 +496,7 @@ Replace the following in the sample script:
...
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/sample-data.md:497]:
```suggestion
  your [database](/influxdb/clustered/admin/databases/)
```

File: content/influxdb/clustered/write-data/line protocol/influxctl-cli.md [hunk 18]
```diff
@@ -67,7 +67,7 @@ to write the [home sensor sample data](#home-sensor-data-line-protocol) to your
 Provide the following:
 
 - Database name using the `--database` flag
-- [Database token](/influxdb/clustered/admin/tokens/) (with write permissions
+- [database token](/influxdb/clustered/admin/tokens/#database-tokens) (with write permissions
```
Comment (jstirnaman) [inline content/influxdb/clustered/write-data/line protocol/influxctl-cli.md:70]:
```suggestion
- A [database token](/influxdb/clustered/admin/tokens/#database-tokens) (with write permissions
```

File: content/influxdb/clustered/write-data/line protocol/influxctl-cli.md [hunk 19]
```diff
@@ -67,7 +67,7 @@ to write the [home sensor sample data](#home-sensor-data-line-protocol) to your
 Provide the following:
 
 - Database name using the `--database` flag
```
Comment (jstirnaman) [inline content/influxdb/clustered/write-data/line protocol/influxctl-cli.md:69]:
```suggestion
- The [database](/influxdb/clustered/admin/databases/) name using the `--database` flag
```

File: content/influxdb/clustered/write-data/line protocol/influxctl-cli.md [hunk 20]
```diff
@@ -67,7 +67,7 @@ to write the [home sensor sample data](#home-sensor-data-line-protocol) to your
...
-- [Database token](/influxdb/clustered/admin/tokens/) (with write permissions
+- [database token](/influxdb/clustered/admin/tokens/#database-tokens) (with write permissions
   on the target database) using the `--token` flag
 - Timestamp precision as seconds (`s`) using the `--precision` flag
```
Comment (jstirnaman) [inline content/influxdb/clustered/write-data/line protocol/influxctl-cli.md:72]:
```suggestion
- The timestamp precision as seconds (`s`) using the `--precision` flag
```

**Review by jstirnaman — APPROVED** [review]
Looks good - a few minor corrections, but mostly suggestions about preexisting inconsistencies.

**Pull request merged**